curated_transformers/models/pytorch/attention.py

```diff
@@ -8,28 +8,37 @@
 from ...errors import Errors
 
 
-@dataclass
```

Comment:
Do we need to add the `@torch.jit.script` decorator like we do for `PyTorchTransformerOutput`?

Reply:
For some reason we don't need it (the quantized models work fine without the annotation). I don't really understand yet why we do need it for `PyTorchTransformerOutput` (it still fails without an annotation on that class).